README.md

```diff
@@ -55,6 +55,8 @@ search these unofficial resources:
 * [Questions tagged 'node.js' on StackOverflow][]
 * [#node.js channel on chat.freenode.net][]. See <http://nodeirc.info/> for more
   information.
+* [Node.js Slack Community](http://node-js.slack.com): Visit
```

Comment:
Micro-nit: There should be a trailing slash on the URL in this line.

Reply:
Oh, much less of a nit and more important: This should be `https`. (Yes, it redirects. But still, that first request can send cookies unencrypted and have MITM content inserted and all that stuff.)